### DISCUSSION TITLE
Markdown-Editor actions

### DISCUSSION THREAD

**Discussion by Dominic-Marcelino** [directus/directus#4389] [Feature Requests]
Currently when you want to remove the semantic tags (eg the two starts for bold) via button or shortcut, you have to mark the text PLUS the tags.

To align with most others editors we should allow to remove them with selecting the text only.

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
